app/containers/App/reducer.js

```diff
@@ -6,9 +6,9 @@
  * add it to the switch statement in the reducer function
  *
  */
-
 import produce from 'immer';
 import { LOAD_REPOS_SUCCESS, LOAD_REPOS, LOAD_REPOS_ERROR } from './constants';
+import { LOGIN_SUCCESS_ACTION, LOGIN_ERROR_ACTION } from '../Login/constants';
```

Comment:
'containers/Login/constants' quando puoi ti consiglio di usare i path assoluti che allentano un po' le dipendenze e semplificano eventuali refactor in futuro.. ad es se in futuro dovessi spostare questo file si romperebbe il link ora.